tests/packagedcode/data/npm/upm/package/package.json.expected

```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "type": "npm",
```

Comment:
@schischi I'm not sure if we can use type as npm here, as these packages are not really present in the main package respository for npm, ie. at npmjs, see https://www.npmjs.com/search?q=com.unity.ml-agents which does not point to any package, so this does not really align with https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#npm possibly? Also from https://docs.unity3d.com/Manual/upm-manifestPkg.html:
```
The file’s format is similar to [npm](https://www.npmjs.com/)’s package.json format, but uses different semantics for some of its properties
```
So this is not really same as npm, just similar? Do you think this should be a new type in the purl-spec instead (like `upm` instead of `npm`)?

what do you think @pombredanne?

Reply:
It should be a `upm` or `unity` thing. These are not npms for sure.